.github/workflows/main.yml

```diff
@@ -62,12 +53,16 @@ jobs:
       - name: Install python dependencies
         run: |
           pip install  --user --upgrade pip
-          pip install tox
+          pip install pre-commit
+          pip install mypy==0.782
```

Comment:
Any specific reason we put the package here? I think normally we write it in[`dev_requirements.txt`](https://github.com/dbt-labs/dbt-redshift/blob/main/dev_requirements.txt#L12). [Link for dbt-core](https://github.com/dbt-labs/dbt-core/blob/main/dev-requirements.txt#L9)